.travis.yml

```diff
@@ -12,7 +12,8 @@ install:
   - pip install coveralls
 script: 
   - tox
-after_success: coveralls
+after_success:
+  coveralls --merge=coverage/coverage-js.json
```

Comment:
Hopefully not. The [docs](https://github.com/coagulant/coveralls-python#multiple-languages-experimental) say that it should merge coveralls-style json with the Python data it presumably reads from .coverage, but I'll have to wait until it works before saying this for sure